libsql-sqlite3/src/vectorInt.h

```diff
@@ -0,0 +1,62 @@
+#ifndef _VECTOR_H
+#define _VECTOR_H
+
+#include "sqlite3.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Objects */
+typedef struct Vector Vector;
+typedef struct DiskAnnIndex DiskAnnIndex;
+
+#define MAX_VECTOR_SZ 16000
```

Comment:
Was the max hard-coded dimensions of 16,000 picked for a particular reason?  Other vector DBs support larger numbers of dimensions (qdrant is 65536, milvus defaults to 32768 and can be increased via configuration)

There are valid use cases for large dimensional vectors, so if there is no specific limitation that the 16,000 is for... could this be made larger?  I would set it to 65536 (uint16 max size) to be in better alignment with the large dimensional vectors that other vector DBs support

Reply:
Fixed in https://github.com/tursodatabase/libsql/pull/1402/commits/469e127ec1145a040d06b19e5a6c8876024f2406, thanks @alex-sherwin!